README.md

```diff
@@ -25,7 +25,23 @@ Basic installation scenario
 public void ConfigureServices(IServiceCollection services)
 {
     // (...)
-    services.AddEpiserverRedirects(options => ...);
+    services.ConfigureEpiserverRedirects(
+        options =>      // configure caching, priority, etc
+        {
+            options.Caching.UrlRedirectCacheEnabled = true;
+            options.PreserveQueryString = false;
+            options.AddAutomaticRedirects = false;
+            options.SystemRedirectRulePriority = 80;
+            options.DefaultRedirectRulePriority = 100;
+        }
+        repoConfig =>   // configure custom rule store here. Skip to fallback to Dynamic Data Store.
+        {
+            repositoryConfig.AddSqlServerRepository("connectionString",
```

Comment:
Let's not add specific repository in this example. Instead let's just link them to separate bullet point in README so that user can read more about various scenarios they can configure (or skip in order to use default DDS).
If I were a user I would like to read this README kind of 'incrementally'
Here I can see example usage of the Configure method, and then I can read more about specific parameter (repositoryConfiguration) and then I have a description of optional step: implementing your own IRedirectRuleRepository.
When writing README always try to be in users' shoes and think what would be helpful read (and in what order) ;)